aten/src/ATen/xpu/profiler/XPUActivityProfiler.cpp

```diff
@@ -0,0 +1,140 @@
+#include <ATen/xpu/profiler/XPUActivityApi.h>
+#include <ATen/xpu/profiler/XPUActivityProfiler.h>
+
+#include <c10/core/DeviceType.h>
+#include <ATen/KinetoPluginAPI.h>
+
+#include <chrono>
+
+namespace at::kineto_plugin::xpu {
+
+uint32_t XPUActivityProfilerSession::iterationCount_ = 0;
+
+// =========== Session Constructor ============= //
+XPUActivityProfilerSession::XPUActivityProfilerSession(
```

Comment:
What about add the **ptiViewGPULocalAvailable** in the constructor of the **XPUActivityProfilerSession**?
The **XPUActivityProfilerSession** will be initialized at the very first place when the torch.profiler is called.
CC: @gujinghui